pikaday.js

```diff
@@ -348,6 +349,19 @@
         if (opts.isEndRange) {
             arr.push('is-endrange');
         }
+
+        opts.startRangeArray.forEach(function(startRange) {
```

Comment:
You could use arrow functions in these, right?

Reply:
Yes! But Pikaday is standalone vanilla javascript plugin. So we have to right es5 in this repo in order to be compatible everywhere